doc/source/examples/incompressible-flow/2d-transient-flow-around-cylinder/2d-transient-flow-around-cylinder.rst

```diff
@@ -0,0 +1,266 @@
+
+.. image:: images/geometry_description.png
+    :alt: The geometry
+    :align: center
+    :name: geometry_description
+
+The flow field features a stable laminar boundary layer at the cylinder leading edge and a recirculation zone behind it formed by two unstable vortices of opposite signs. These vortices successively detach from the cylinder in a periodic manner (vortex shedding), leading to the generation of the von Kármán vortex street pattern in the wake. This vortex shedding causes a fluctuating pressure force acting on the cylinder, resulting in oscillations of the drag and lift coefficients in time. The frequency of vortex shedding is related to the Strouhal number:
+
+.. math::
+ S_t = \frac{D f_v}{U_\infty}
+
+where :math:`D` is the diameter of the cylinder, :math:`f_v` is the frequency of the shedding and :math:`U_\infty` is the upstream velocity.
+
+Parameter file
+--------------
+
+Simulation control
+~~~~~~~~~~~~~~~~~~
+This example uses a 2nd order backward differentiation (``method = bdf2``) for the time integration scheme. The simulation time is set to 200 seconds with the ``time end`` parameter and a time step of 0.05 seconds is used (``time step = 0.05``).
+
+.. code-block:: text
+
+    subsection simulation control
+        set method                = bdf2
+        set output name           = cylinder-output
+        set output frequency      = 1
+        set output path           = ./Re200/
+        set time end              = 200.0
+        set time step		          = 0.05
+        set subdivision           = 1
+    end
+
+FEM interpolation
+~~~~~~~~~~~~~~~~~
+
+The interpolation orders for the velocity and pressure are set to Q2-Q1 in the ``FEM`` subsection:
+
+.. code-block:: text
+
+    subsection FEM
+        set velocity order  = 2
+        set pressure order  = 1
+    end
+
+Mesh
+~~~~~
+
+The initial mesh is generated with `Gmsh <https://gmsh.info/#Download>`_ and imported as described in  `2D Flow around a cylinder <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.html>`_.
+
+.. code-block:: text
+
+    subsection mesh
+        set type                = gmsh
+        set file name           = cylinder_structured.msh
+        set initial refinement  = 1
+    end
+
+Mesh adaptation control
+~~~~~~~~~~~~~~~~~~~~~~~
+
+While the discretization in the wake of the cylinder has less impact on the forces acting on the cylinder wall than the boundary layer discretization, it is interesting to well resolve the wake to capture the von Kármán vortex street pattern. Therefore, to adapt the mesh in the boundary layer and the wake as the vortices are shed, a non-uniform mesh adaptation is performed at each time step and the parameters are specified using the ``Mesh Adaptation Control`` subsection:
+
+.. code-block:: text
+
+   subsection mesh adaptation
+       set type                   = kelly
+       set variable               = pressure
```

Comment:
It would be better if you could add a sentence to explain why you choose pressure as the refinement parameter.

Reply:
I added : 

`we have observed that the refinement has less tendency to follow the vortices as they move through the wake with the pressure as the refinement indicator than if we select the velocity.`

 is this better?